daemon/config.go

```diff
@@ -206,6 +212,34 @@ func getConflictFreeConfiguration(configFile string, flags *flag.FlagSet) (*Conf
 			return nil, err
 		}
 
+		// make sure that values set in the config file with nullable values, like `false`,
+		// are not overriden by default truthy values from the flags that were not explicitly set.
```

Comment:
Looking further, findConfigurationConflicts seems to be meant as a read-only/error reporting function that can also be called independent of getConflictFreeConfiguration. So its fair that you dont want to merge them.